src/hotspot/cpu/x86/stubGenerator_x86_32.cpp

```diff
@@ -4130,11 +3914,7 @@ class StubGenerator: public StubCodeGenerator {
     StubRoutines::x86::_d2i_wrapper                = generate_d2i_wrapper(T_INT,  CAST_FROM_FN_PTR(address, SharedRuntime::d2i));
     StubRoutines::x86::_d2l_wrapper                = generate_d2i_wrapper(T_LONG, CAST_FROM_FN_PTR(address, SharedRuntime::d2l));
 
-    // Build this early so it's available for the interpreter
-    StubRoutines::_throw_StackOverflowError_entry          = generate_throw_exception("StackOverflowError throw_exception",
-                                                                                      CAST_FROM_FN_PTR(address, SharedRuntime::throw_StackOverflowError));
-    StubRoutines::_throw_delayed_StackOverflowError_entry  = generate_throw_exception("delayed StackOverflowError throw_exception",
-                                                                                      CAST_FROM_FN_PTR(address, SharedRuntime::throw_delayed_StackOverflowError));
+      CAST_FROM_FN_PTR(address, SharedRuntime::throw_delayed_StackOverflowError));
```

Comment:
This does not seems correct. And I see linux-x86 (32 bits) corresponding build failure in GHA.

Reply:
Sorry, that was an accidental paste while editing the code. Should be fixed now.